### ISSUE TITLE
Can't install plai-typed

### ISSUE THREAD

**Issue by andres-moreno** [mflatt/plai-typed#17]
Tried to install `plai-typed` via `raco pkg install` and got an error message similar to [this uu-cs510 error message](https://pkg-build.racket-lang.org/server/built/fail/uu-cs5510.txt).

When I looked at the [package index](https://pkgd.racket-lang.org/pkgn/search?q=scheme-doc), I see that the package failed to build. Clicking on the [to do](https://pkg-build.racket-lang.org/server/built/fail/scheme-doc.txt) shows the same type of error as above.

Work-around: downgraded from Racket 8.6 to Racket 8.5 and was able to get `plai-typed` to install.

**Comment by adamsmd** [comment]
I am seeing the same error with Racket 8.2, though I was able to get it working with `--force` (I'm sure that is a huge bodge, but I just needed some way to test out the language).

```
$ racket --version
Welcome to Racket v8.2 [cs].

$ raco pkg install plai-typed
Resolving "plai-typed" via https://download.racket-lang.org/releases/8.2/catalog/
Resolving "plai-typed" via https://pkgs.racket-lang.org
Using cached16659967911665996791289 for github://github.com/mflatt/plai-typed/master
The following uninstalled packages are listed as dependencies of plai-typed:
   scheme-doc
Would you like to install these dependencies? [Y/n/a/c/?] y
Resolving "scheme-doc" via https://download.racket-lang.org/releases/8.2/catalog/
Resolving "scheme-doc" via https://pkgs.racket-lang.org
Using cached16659968141665996814697 for https://github.com/racket/racket.git?path=pkgs%2Fscheme-doc
raco pkg install: packages conflict
  package: scheme-doc
  package: racket-doc
  module path: (lib "scribblings/scheme/compat.scrbl")
```

**Comment by adamsmd** [comment]
After @mflatt updated the repository, this seems to be fixed, and it works for me now on v8.2.

I recommend closing this issue as fixed.

**Comment by andres-moreno** [comment]
Thanks for fixing this--I really appreciate it!
